core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -175,8 +176,8 @@ public FSDataOutputStream create(Path path, FsPermission permission, boolean ove
     }
 
     AlluxioURI uri = new AlluxioURI(HadoopUtils.getPathWithoutScheme(path));
-    CreateFilePOptions options = FileSystemClientOptions.getCreateFileOptions().toBuilder()
-        .setBlockSizeBytes(blockSize).setMode(permission.toShort()).build();
+    CreateFilePOptions options = CreateFilePOptions.newBuilder().setBlockSizeBytes(blockSize)
+        .setMode(permission.toShort()).setRecursive(true).build();
```

Comment:
why is recursive set now?

Reply:
Alluxio Java API change. Explained in other comment.